torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -9725,6 +9725,47 @@ def forward(self, inp):
                 ddp._check_reducer_finalized()
                 ddp(input)
 
+        @skip_if_lt_x_gpu(2)
+        @skip_but_pass_in_sandcastle_if(
+            BACKEND not in DistTestCases.backend_feature["ddp"],
```

Comment:
Is this guaranteed to be NCCL-only, or is it expected that all backends would pass this test?